pkg/controller/router.go

```diff
@@ -0,0 +1,31 @@
+package controller
+
+import (
+	"log"
+	"net/http"
+
+	"luis/goapi/pkg/service"
+
+	"github.com/gin-gonic/gin"
+)
+
+func Start() {
+
+	r := gin.Default()
```

Comment:
this is the router, and should go on it's own packages, controller should only contain the handlers